VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/participant-waiting-room.component.ts

```diff
@@ -191,7 +195,20 @@ export class ParticipantWaitingRoomComponent extends WaitingRoomBaseComponent im
     }
 
     getPrivateConsultationParticipants(): Participant[] {
-        return this.conference.participants.map(p => new Participant(p)).filter(p => !p.isJudge && p.id !== this.participant.id);
+        return this.conference.participants
+            .filter(
+                p =>
+                    p.id !== this.participant.id &&
+                    p.role !== Role.JudicialOfficeHolder &&
+                    p.role !== Role.Judge &&
+                    p.hearing_role !== HearingRole.OBSERVER &&
+                    p.hearing_role !== HearingRole.WITNESS
+            )
+            .map(p => new Participant(p));
```

Comment:
Is this map() gonna cause an issue again?

Reply:
lol
this is the map youve removed in ur PR :)